app/questionnaire/placeholder_transforms.py

```diff
@@ -41,8 +47,29 @@ def __init__(
 
     input_date_format = "%Y-%m-%d"
 
-    def format_currency(self, number: Decimal | float, currency: str = "GBP") -> str:
-        formatted_currency: str = format_currency(number, currency, locale=self.locale)
+    def format_currency(
+        self,
+        number: int | Decimal | float,
+        currency: str = "GBP",
+        decimal_limit: int | None = None,
+        raw_args: Mapping | None = None,
+    ) -> str:
+        if raw_args:
```

Comment:
We need to doc string what raw args are used for and when they might not exist etc.